students/DanCornutt/session06/test_mailroom.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import os
+import pytest
+from mailroom import *
+
+
+def test_check_donor_1():
+    """Finds donor in list"""
+    assert check_donor("Billy Bills") is True
+
+def test_check_donor_2():
+    """Does not find donor in list"""
+    assert check_donor("Billy Billz") is False
+
+def test_gen_letter_1():
+    """Generates valid letter for donor in list"""
+    assert gen_letter("Billy Bills") == """Dearest Billy Bills,
+
+    We greatly thank you for your recent contribution of $300.00.
+
+    It will go straight to the person who needs it the most, our CEO.
+
+    Please give more next time.
+
+    	Love,
+
+    		The Team"""
+
+def test_gen_letter_1():
+    """Does not generate valid letter for donor not in list"""
+    with pytest.raises(KeyError):
```

Comment:
note that this is going to pass or fail based on a KeyError or not being raised -- so you don't need to actually check anything about the result.